hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -361,7 +361,7 @@ public void testRollbackOperations(HoodieTableType tableType) throws Exception {
   @Test
   public void testManualRollbacks() throws Exception {
     HoodieTableType tableType = COPY_ON_WRITE;
-    init(tableType, false);
+    init(tableType, true);
```

Comment:
why this change? this test intentionally not enabled at the beginning and enables at line 377.

Reply:
fixed.